packages/bpk-component-input/src/BpkInput.js

```diff
@@ -34,7 +37,12 @@ export const INPUT_TYPES = {
 
 const BpkInput = props => {
   const classNames = [getClassName('bpk-input')];
+  const containerClassNames = [getClassName('bpk-input--clearable__container')];
+  const clearButtonClassNames = [
+    getClassName('bpk-input--clearable__clear-button'),
+  ];
   const {
+    value,
```

Comment:
if you get the chance to order them alphabetically would make your code, undoubtedly beautiful, even better!